SofaKernel/modules/SofaBaseVisual/InteractiveCamera.cpp

```diff
@@ -36,7 +36,6 @@ SOFA_DECL_CLASS(InteractiveCamera)
 
 int InteractiveCameraClass = core::RegisterObject("InteractiveCamera")
         .add< InteractiveCamera >()
-        .addAlias("Camera")
```

Comment:
Will this be breaking for user scenes?

Reply:
Yes. A scene with "Camera" in it will now have the Camera object and not the InteractiveCamera. 

The easy answer: 
We are changing an alias and that this alias is not used at all in any sofa examples I doubt anyone knows it even exists. In addition it is displayed as InteractiveCamera in the GUI so ...well the hardcoded alias are flawned by design so removing one is a good thing :) 

The real answer: 
The question is how could we warn users that something has changed. 
One option could be to use the <APIVersion> for that. I have no clear view on how to do that but we can probably use it control which version the string "Camera" is pointing to. 
The other option I see would be to improve the factory so that it can group components from the same plugin to implement namespace in order to avoid name clash or explicit control. 
Eg:
```xml 
<Import plugin="SofaVisual>
<SofaVisual.Camera/>
 
<From plugin="SofaVisual" import="Camera"/>     <= Here it is very clear that Camera has "overriden the old camera object">
<Camera/>
```

Reply:
Shame on me, I forgot about the SceneCheckAPIChange which would allow a very easy way to implement the needed warnings on the change. I will do that.